nodejs_install.sh

```diff
@@ -13,14 +13,15 @@ else
   curl -fsSL https://deb.nodesource.com/setup_19.x | sudo -E bash - 
 fi
 
-# Install NodeJS and NPM
+# Install NodeJS, NPM, and update browserslist-db@latest
 if ( which node ) 
 then
   echo -e "\n==== NodeJS installed ====\n"
 else 
-  echo -e "\n==== Installing NodeJS && NPM ====\n"
+  echo -e "\n==== Installing NodeJS && NPM && Update Browserlist ====\n"
   sudo apt install -y nodejs  
   sudo apt install -y npm
+  sudo npx update-browserslist-db@latest
```

Comment:
Why not just add this to the package.json?

Reply:
Added to package.json.